src/components/manage/Widgets/biogeographical.js

```diff
@@ -0,0 +1,27 @@
+export const biogeographicalData = [
```

Comment:
I think the values here are the same as labels (Title case), at least this is what we get from the RestAPI.

On the other hand, if we lock to plone_registry, we have region geonames codes as key:

[biogroups.xml.txt](https://github.com/eea/volto-widget-geolocation/files/5347167/biogroups.xml.txt)

Still, as the old code uses the `label` to fill the `location` metadata, we don't have to use the geonames codes, either.